src/parsing/scope.ml

```diff
@@ -508,8 +508,15 @@ let scope_term :
       ?find_sym:find_sym -> ?typ:bool -> ?mok:(int -> meta option)
       -> bool -> sig_state -> env -> p_term -> term =
   fun ?find_sym ?(typ=false) ?(mok=fun _ -> None) m_term_prv ss env t ->
-  let md = M_Term {m_term_meta_of_key=mok; m_term_prv} in
-  Bindlib.unbox (scope ?find_sym ~typ 0 md ss env t)
+  try
+    let md = M_Term {m_term_meta_of_key=mok; m_term_prv} in
+    Bindlib.unbox (scope ?find_sym ~typ 0 md ss env t)
+  with Fatal(Some p,    msg) ->
+    Console.out 3 (Color.red "[%a]") Pos.pp p;
+    (**  Get ride of the position because the error message is displayed here
+        There is no more need of the position and this avoids the message
+        to be displayed twice in [Error.handle_exceptions] *)
+    raise(Fatal(None,    msg))
```

Comment:
Actually, it is not that the printed position is the one of the proof instead of the symbol y. Rather, it Is just that the position of symbol y is not printed. Thus, the position of the proof is printed (because of verbose level 3) just above the error message which makes the user think that the position printed is the one where the error occurred.

Reply:
The proposed fix, just prints the position of the symbol before raising the exception. Removing the position from the exception avoids to show the position twice when the check is invoked from the command line.

Reply:
Notice, I could also print the message in sig_state.ml find_sym, as soon as the symbol is not found line118.
But it seemed to me more natural to it here.

Reply:
It would be more appropriate in Sig_state.find_sym. But because this problem does not occur in command line, it seems that the problem is rather in the lsp server. So there should be no modification in lambdapi but in the lsp server only. Perhaps the fatal error is not added in the logs. Could you please check this?

Reply:
Ok. The fatal error is added in Pure.handle_tactic after the try-with. So I can print in the location just before. That would avoid changing Lambdapi. 
I will push the modification and you can check if it is better that way.